src/main/java/org/sasanlabs/service/vulnerability/openRedirect/Http3xxStatusCodeBasedInjection.java

```diff
@@ -81,10 +81,11 @@ public ResponseEntity<?> getVulnerablePayloadLevel1(
     // Payloads:
     // 1. Protocol other than http can be used e.g. ftp://ftp.dlptest.com/ also
     // 2. "//facebook.com"
+
     @AttackVector(
             vulnerabilityExposed = {VulnerabilityType.OPEN_REDIRECT_3XX_STATUS_CODE},
             description =
-                    "OPEN_REDIRECT_QUERY_PARAM_DIRECTLY_ADD_TO_LOCATION_HEADER_IF_NOT_HTTP_HTTPS_OR_DOMAIN_IS_SAME")
+                    "OPEN_REDIRECT_QUERY_PARAM_DIRECTLY_ADD_TO_LOCATION_HEADER_IF_NOT_HTTP_HTTPS_WWW_OR_DOMAIN_IS_SAME")
```

Comment:
Ehis, we need to update in:

1. https://github.com/SasanLabs/VulnerableApp/blob/master/src/main/resources/i18n/messages.properties
2. https://github.com/SasanLabs/VulnerableApp/blob/master/src/main/resources/i18n/messages_en_US.properties
as well. Actually these strings are locale specific strings so if we don't update them then we will not be able to run the project. I would suggest to please build and run the project after the change to see if everything is running as expected.